android/timodule.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ti:module xmlns:ti="http://ti.appcelerator.org" xmlns:android="http://schemas.android.com/apk/res/android">
-	<!--
-		Similar to tiapp.xml, but contains module/platform specific
-		configuration in <iphone> and <android> sections
-	-->
 	<iphone>
 	</iphone>
 	<android xmlns:android="http://schemas.android.com/apk/res/android">
+		<manifest>
+			<application>
+        		<activity android:name="ti.stripe.TiStripeHostActivity" android:configChanges="keyboard|keyboardHidden|screenLayout|screenSize|orientation" android:exported="true" android:label="Stripe" />
```

Comment:
```suggestion
        		<activity android:name="ti.stripe.TiStripeHostActivity"  android:theme="@android:style/Theme.Translucent.NoTitleBar"  android:configChanges="keyboard|keyboardHidden|screenLayout|screenSize|orientation" android:exported="true" android:label="Stripe" />
```

will make the background of the window transparent and not fully white/black

Reply:
Thank you!